build_library/toolchain_util.sh

```diff
@@ -194,7 +194,9 @@ get_cross_pkgs() {
 # Get portage arguments restricting toolchains to binary packages only.
 get_binonly_args() {
     local pkgs=( "${TOOLCHAIN_PKGS[@]}" $(get_cross_pkgs "$@") )
-    echo "${pkgs[@]/#/--useoldpkg-atoms=}" "${pkgs[@]/#/--rebuild-exclude=}"
+    # XXX: Revert this binutils change after stable > 1535.
+    local tmppkgs=( "${TOOLCHAIN_PKGS[@]#sys-devel/binutils*}" $(get_cross_pkgs "$@") )
```

Comment:
I think just deleting or commenting out the line in the TOOLCHAIN_PKGS variable would be more readable. Would that not work for some reason?

Reply:
This doesn't work at all yet for amd64 in automated builds.  I am still working on it.